cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -545,6 +560,13 @@
     }
 }
 
+.no-js .o-mega-menu {
```

Comment:
no, I mean have classes that `show/hide` depending on if `no-js` is included on the body.